### PR TITLE
arm64: dts: qcom: Add dt for fortuna3g

### PR DESCRIPTION
**Pull request by SiddharthManthan**
This pull request adds support for fortuna3g device.

Add initial support for :
* gpio keys and WCNSS
* Vibrator
* rt5033 fuelgauge. Also compatible with sec-fuelgauge
* Zinitix BT541 with touchkey
* Modem and sound
* Audio jack

The following pull requests need to merged first :
- [ ] Update Kernel to 5.17
- [ ] https://github.com/msm8916-mainline/lk2nd/pull/151
- [ ] https://github.com/msm8916-mainline/linux-panel-drivers/pull/10
- [ ] https://github.com/msm8916-mainline/linux/pull/222

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/msm8916-samsung-fortuna3g.dts [hunk 1]
```diff
@@ -11,11 +11,32 @@
 
 	reserved-memory {
 		/* Additional memory used by Samsung firmware modifications */
-		tz-apps@85a00000 {
```
Comment (TravMurav) [inline arch/arm64/boot/dts/qcom/msm8916-samsung-fortuna3g.dts]: this change seem to be in a wrong commit

File: drivers/gpu/drm/panel/msm8916-generated/Makefile [hunk 2]
```diff
@@ -11,9 +11,9 @@ obj-$(CONFIG_DRM_PANEL_MSM8916_GENERATED) += panel-motorola-harpia-boe.o
 obj-$(CONFIG_DRM_PANEL_MSM8916_GENERATED) += panel-motorola-harpia-tianma.o
 obj-$(CONFIG_DRM_PANEL_MSM8916_GENERATED) += panel-motorola-osprey-inx.o
 obj-$(CONFIG_DRM_PANEL_MSM8916_GENERATED) += panel-motorola-surnia-boe.o
-obj-$(CONFIG_DRM_PANEL_MSM8916_GENERATED) += panel-samsung-ea8061-ams549bu19-id400418.o
```
Comment (wonderfulShrineMaidenOfParadise) [inline drivers/gpu/drm/panel/msm8916-generated/Makefile:14]: Same here.

File: arch/arm64/boot/dts/qcom/Makefile [hunk 3]
```diff
@@ -27,8 +27,7 @@ dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-mtp.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-samsung-a3u-eur.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-samsung-a5-zt.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-samsung-a5u-eur.dtb
-dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-samsung-e7.dtb
```
Comment (wonderfulShrineMaidenOfParadise) [inline arch/arm64/boot/dts/qcom/Makefile:30]: Well my E7 is just merged please don't drop it  so quickly :p

**Review by wonderfulShrineMaidenOfParadise — CHANGES_REQUESTED** [review]

**Comment by stephan-gh** [comment]
Closing in favor of https://github.com/msm8916-mainline/linux/pull/227 (I guess)

**Pull request closed without merging**